test/runtests.sh

```diff
@@ -20,10 +20,10 @@ then
 fi
 
 # run the program, feed commands from input.txt file and redirect the output to the actual.txt
-java -classpath ../bin seedu.addressbook.Main < input.txt > actual.txt
+java -classpath ../bin seedu.addressbook.Main < sortinput.txt > actual.txt
```

Comment:
Why do you have a separate sortinput.txt instead of incorporating the usual input.txt?

Reply:
i feel its easier to do 2 different separate test case files
1)because if there is a error that causes sortinput.txt to be deleted/altered due to not ending the program, you can still end the program not worrying too much on it because you can copy it easily.
2) i am a windows 10 user who is used to using windows10 shortcuts access to go to recent files, you just have to do this in windows10 if you want to use the shortcuts access( unless there is a way to highlight or something which i dont know). i will get 2 input.txt from my search and i cant tell the difference.
There are many ways to solve this simple problem(save the original test cases in somewhere else) but i generally do this.

Reply:
hmm I'm not too sure I understand your reasons @souless94, you can clarify them with me during tutorial if you like. However, I feel that you should edit the original file because in the PR it would be clearer to see which lines you have changed. In this case where you have created a new file, all the lines in the new file are highlighted in green and it's hard to find which statements you have added to test your sort command.

Reply:
Ok noted, i will edit original file in the future. Its more of a habit of mine, because
1) it can save time for typos in code that causes error, i know right now its working properly ( eg, like if you are in wrong directory, file not found exception can be shown, and you know you are in wrong folder... ) 
2)working with many input.txt files and i am lazy to navigate to the folder so i use search in windows haha. (main reason)